tests/unit/modules/test_junos.py

```diff
@@ -954,6 +956,51 @@ def test_cli_exception_in_cli(self):
             ret["out"] = False
             self.assertEqual(junos.cli("show version"), ret)
 
+    def test_cli_output_save(self):
```

Comment:
Adjust tests for restored interfaces, for example: diff, rpc, etc.

Reply:
not needed now as we reverted interface params